playbook-website/app/helpers/changelog_helper.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module ChangelogHelper
+  def changelog_to_hash(changelog)
+    posts = []
+
+    markdown_html = render_markdown(changelog)
+    document = parse_html(markdown_html)
+
+    document.css("h1").lazy.take(5).each_with_index do |title_element, index|
+      title_text = title_element.text
+      puts title_text
+      ### Only create a post of the title has letters. Small releases only have a version number as thier title
+      if title_text.match?(/[a-zA-Z]/)
+        post = extract_post_data(document, title_element, index)
+        posts << post
+      end
```

Comment:
Is this okay? i have it where it only creates a post if a release title has letters. im assuming that when the title is just a release number, it will not have an image, therefor we can skip that post??